cves/CVE-2003-0020.yml

```diff
@@ -48,10 +54,8 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
    - commit: 88f261355a83c28097a18f9b15d92196ec290def
-     note:
+     note: Added input validation
```

Comment:
Did it only check for the escape sequence? Was there any other sequences it started to check for?

Reply:
In terms of input validation, the fix checked for a few different kinds of escape sequences, probably specifically the ones most likely to cause problems if exploited.